File: core/NRepeatedElementInSize2nArray/NRepeatedElementInSize2nArray.py
from typing import List
import random


def repeatedNTimes(nums: List[int]) -> int:
    while True:
        x, y = random.randint(0, len(nums) - 1), random.randint(0, len(nums) - 1)
        if x != y and nums[x] == nums[y]:
            return nums[x]

File: core/NRepeatedElementInSize2nArray/NRepeatedElementInSize2nArray.java
package com.dbc;

import java.util.Random;

public class NRepeatedElementInSize2nArray {
    public int repeatedNTimes(int[] nums) {
        Random random = new Random();
        while (true) {
            int x = random.nextInt(nums.length);
            int y = random.nextInt(nums.length);
            if (x != y && nums[x] == nums[y]) return nums[x];
        }
    }
}
